x/rollapp/keeper/invariants.go

```diff
@@ -138,6 +160,29 @@ func BlockHeightToFinalizationQueueInvariant(k Keeper) sdk.Invariant {
 					broken = true
 				}
 			}
+
```

Comment:
maybe also check that `stateInfo` is pending? 
```stateInfo.Status = commontypes.Status_PENDING```

Reply:
it's already checked in another invariant